src/helpers/theme.js

```diff
@@ -1,223 +1,248 @@
+        backgroundColor: backgroundColor,
+      },
+      item: {
+        padding: 10,
+      },
+    },
     table: {
-      color: "#006273",
-    },
-    actions: {
-      color: "#006273",
-    },
-    header: {
-      color: "#006273",
-      align: "center",
-    },
-  },
-  dialog: {
-    title: {
-      fontWeight: 500,
-      color: "grey",
-    },
-    content: {
-      padding: 0,
-    },
-    primaryButton: {
-      backgroundColor: "#006273",
-      color: "#fff",
-      fontWeight: "bold",
-      "&:hover": {
-        backgroundColor: alpha("#006273", 0.5),
-        color: "#006273",
-      },
-    },
-    secondaryButton: {},
-  },
-  tooltipContainer: {
-    position: 'fixed',
-    bottom: 15,
-    right: 8,
-    zIndex: 2000,
-    display: 'flex',
-    flexDirection: 'column',
-    alignItems: 'flex-end',
-  },
-  flexTooltip: {
-    marginBottom: 5,
-  },
-  fab: {
-    position: "fixed",
-    bottom: 20,
-    right: 8,
-    zIndex: 2000,
-  },
-  fakeInput: {},
-  bigAvatar: {
-    width: 160,
-    height: 160,
-  },
-  buttonContainer: {
-    horizontal: {
+      title: {
+        padding: 10,
+        fontWeight: 500,
+        color: primaryColor,
+        backgroundColor: headerColor,
+      },
+      header: {
+        color: primaryColor,
+      },
+      headerAction: {
+        padding: 5,
+      },
+      row: {
+        color: primaryColor,
+        align: "center",
+        "&:hover": {
+          background: hoveredTableRowColor + " !important",
+        },
+      },
+      cell: {
+        padding: 5,
+      },
+      lockedRow: {
+        background: lockedBackgroundPattern,
+      },
+      lockedCell: {},
+      highlightedRow: {},
+      highlightedCell: {
+        fontWeight: 500,
+        align: "center",
+      },
+      highlightedAltRow: {},
+      highlightedAltCell: {
+        fontStyle: "italic",
+        align: "center",
+      },
+      disabledRow: {},
+      disabledCell: {
+        color: greyColor,
+        align: "center",
+      },
+      footer: {
+        color: primaryColor,
+      },
+      pager: {
+        color: primaryColor,
+      },
+    },
+    form: {
+      spacing: 10,
+    },
+    formTable: {
+      table: {
+        color: primaryColor,
+      },
+      actions: {
+        color: primaryColor,
+      },
+      header: {
+        color: primaryColor,
+        align: "center",
+      },
+    },
+    dialog: {
+      title: {
+        fontWeight: 500,
+        color: greyColor,
+      },
+      content: {
+        padding: 0,
+      },
+      primaryButton: {
+        backgroundColor: primaryColor,
+        color: whiteColor,
+        fontWeight: "bold",
+        "&:hover": {
+          backgroundColor: alpha(primaryColor, 0.5),
+          color: primaryColor,
+        },
+      },
+      secondaryButton: {},
+    },
+    tooltipContainer: {
+      position: "fixed",
+      bottom: 15,
+      right: 8,
+      zIndex: 2000,
       display: "flex",
-      flexWrap: "nowrap",
```

Comment:
any reason to remove nowrap ?

Reply:
The same as prev comment. This is still under 'buttonContainer' as it is on develop. Git 'diff' shows this in an inappropriate misleading form.